src/components/document-import-table.js

```diff
@@ -1,7 +1,7 @@
 import by from 'thenby';
 import { Table } from 'antd';
 import PropTypes from 'prop-types';
-import { getDocUrl } from '../utils/urls.js';
+import urls from '../utils/urls.js';
```

Comment:
Bonus commit here:
- changed `urls` usage towards most common usages and adjusted exports accordingly